scripts/update-models.ts

```
@@ -0,0 +1,103 @@
+  writeModelFile('src/deepSeek/modelDeepSeek.ts', 'ModelDeepSeek', ids);
+}
+
+async function updatePerplexity(apiKey: string) {
+  const client = new OpenAI({ apiKey, baseURL: 'https://api.perplexity.ai', dangerouslyAllowBrowser: true });
+  const list = await client.models.list();
+  const ids = list.data.map((m: any) => m.id as string);
+  writeModelFile('src/perplexity/modelPerplexity.ts', 'ModelPerplexity', ids);
+}
+
+async function updateGrok(apiKey: string) {
+  const client = new OpenAI({ apiKey, baseURL: 'https://api.x.ai/v1', dangerouslyAllowBrowser: true });
```

ESLint

[error] Replace `·apiKey,·baseURL:·'https://api.x.ai/v1',·dangerouslyAllowBrowser:·true` with `⏎····apiKey,⏎····baseURL:·'https://api.x.ai/v1',⏎····dangerouslyAllowBrowser:·true,⏎·`

(prettier/prettier)